src/bip39.cpp

```diff
@@ -36,26 +36,30 @@ SecureString CMnemonic::Generate(int strength)
     if (strength % 32 || strength < 128 || strength > 256) {
         return SecureString();
     }
-    uint8_t data[32];
-    GetRandBytes(data, 32);
+    SecureVector data;
+    data.resize(32);
```

Comment:
Why not do this in one line: 
`
SecureVector data(32);
` ?

Same comment for other functions.